index.html

```diff
@@ -232,5 +229,16 @@ <h3 class="lets-connect">LET'S CONNECT</h3>
                 </div>
             </div>
         </section>
+    <section class="contact">
+            <h2>Contact Me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xgeqpkgk" method="post">
+            <input type="text" name="text" id="text" placeholder="Name" maxlength="30" required>
+            <input type="email" name="email" id="email" placeholder="Email" required>
+            <textarea name="formMessage" id="message" placeholder="Message" maxlength="500" required></textarea>
```

Comment:
- [ ] to match the Figma design kindly reduce padding for all of your inputs, and try to align the bottom bar in the center

Your Design           | Figma
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/199440709-725f30a4-244b-4cec-b427-4490b7829282.png) | ![image](https://user-images.githubusercontent.com/89282221/199440970-69fda995-f97f-4164-a8f8-b64be20af7df.png)